source/reference/verification.txt

```diff
@@ -0,0 +1,123 @@
+
+====================
+Verify Data Transfer
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Following a sync, verify the successful transfer of your data before you switch
+your application from using the source cluster to the destination cluster. You
```

Comment:
```suggestion
your application to using the destination cluster. You
```

Reply:
I don't want to confuse users into thinking that writes on the source cluster should be continued after commit before verification

Reply:
Fixed